docker_local_dev/mongoDB/Dockerfile

```diff
@@ -0,0 +1,28 @@
+services:
+  mongo:
+    image: mongo:7.0
+    container_name: mongo
+    restart: always
+    ports:
+      - "27017:27017"
```

Comment:
Is there a reason to expose the DB port? If only the bot accesses it, it should be using container networking. Or is this intended only for local development?